README.md

```diff
@@ -1,12 +1,7 @@
 # Alteryx Promote Python Client
-Python library for deploying models built using Python to Alteryx Promote.
-
-### Installation
-```
-pip install promote
-```
-### Example Models
+Library for deploying models built using Python to Alteryx Promote
```

Comment:
I updated this here and on R client to simply say "Library for deploying Python models to Alteryx Promote."

Reply:
Sounds good